README.md

```diff
@@ -99,7 +110,9 @@ $("button").on("click", function(event){
 
 Your Answer:
 ```js
-// write code here
+document.querySelectorAll('button').addEventListener('click', function(event){
+  event.target.style.color = 'red';
+})
 ```
```

Comment:
You'll want to use `querySelector` instead of `querySelectorAll`. The former will return one DOM object, while the latter will return a group of them, regardless of whether there's one or multiple on the page. In this case, you can only call an event listener on an individual DOM object.

If you do have a collection of DOM objects you want to add an event listener to, you could do that using a loop. Give that a shot...